var/spack/repos/builtin/packages/relion/0003-Repair-DoublePrec_CPU-OFF-build-as-reported-by-Filip.patch

```diff
@@ -1,26 +1,11 @@
-From 2daa7447c1c871be062cce99109b6041955ec5e9 Mon Sep 17 00:00:00 2001
-From: Takanori Nakane <nakane.t@gmail.com>
-Date: Thu, 29 Sep 2022 17:10:09 +0900
-Subject: [PATCH] Repair DoublePrec_CPU=OFF build (as reported by @FilipeMaia).
-
----
- src/ml_optimiser.cpp | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/src/ml_optimiser.cpp b/src/ml_optimiser.cpp
-index b1148ceb..5643b092 100644
---- a/src/ml_optimiser.cpp
-+++ b/src/ml_optimiser.cpp
-@@ -6491,7 +6491,7 @@ void MlOptimiser::precalculateShiftedImagesCtfsAndInvSigma2s(bool do_also_unmask
-
+--- b/relion-4.0.0/src/ml_optimiser.cpp	2022-09-28 01:20:54.000000000 +0000
++++ relion-4.0.0/src/ml_optimiser.cpp	2023-06-14 11:17:55.560744380 +0000
```

Comment:
This also seem to change the dirname, in a way that makes the patch and previous state inconsistent with the number of slashes. Can we check if the issue was just with "levels" and, if not, fix the patch above?

Reply:
Or even better, just do 
```
patch("https://github.com/3dem/relion/commit/2daa7447c1c871be062cce99109b6041955ec5e9.patch?full_index=1", sha256="4995b0d4bc24a1ec99042a4b73e9db84918eb6f622dacb308b718146bfb6a5ea")
```

Reply:
Done. Thanks for the suggestion.